docs/mappings/moqo_gbfs_2.3_mapping.md

```diff
@@ -0,0 +1,199 @@
+    * [system_information.json](#system_informationjson)
+    * [vehicle_types.json](#vehicle_typesjson) *(added in v2.1)*
+    * [station_information.json](#station_informationjson)
+    * [station_status.json](#station_statusjson)
+    * [free_bike_status.json](#free_bike_statusjson)
+    * [system_hours.json](#system_hoursjson)
+    * [system_calendar.json](#system_calendarjson)
+    * [system_regions.json](#system_regionsjson)
+    * [system_pricing_plans.json](#system_pricing_plansjson)
+    * [system_alerts.json](#system_alertsjson)
+    * [geofencing_zones.json](#geofencing_zonesjson) *(added in v2.1)*
+
+## Introduction
+
+This specification describes the mapping of MOQO's API to GBFS.
+
+
+## General Information
+
+This section gives a high level overview of the sharing provider's API and the defined mapping to GBFS.
+
+Official API documentation is access restricted, hence not included.
+
+Examples from th
+
+
+## Open Issues or Questions
+
+Open questions are managed as [issues](https://github.com/mobidata-bw/x2gbfs/issues?q=is%3Aissue+is%3Aopen+label%moqo). They should be tagged with `moqo`.
+
+
+## Files
+
+### gbfs.json
+
+Standard file, feed will be provided in feed language `de` only.
+
+### gbfs_versions.json
+
+Optional file, will not be provided. Only GBFSv2.3 supported.
+
+### system_information.json
+
+System information is manually extracted from the providers homepage. Only the real configuration in e.g. [config/stadtwerk-tauberfranken.json](../../config/stadtwerk-tauberfranken.json) is relevant.
+
+
+### vehicle_types.json
+
+Vehicle Types, vehicles are extracted from the `cars` endpoint, extracting this from the returned cars.
+
+
+#### Field Mappings
+
+GBFS Field | Mapping
+--- | ---
+`vehicle_type_id` |  lowercased `row['car_model_name']`, spaces transformed in underscore
+`form_factor` | `row['vehicle_type']` mapped to GBFS vehicle types.
+`rider_capacity`| -
+`cargo_volume_capacity` | -
+`cargo_load_capacity` | -
+`propulsion_type` | `row['fuel_type']` mapped to GBFS vehicle types.
+`eco_label` | -
+`max_range_meters` | Not availabe via API, set to `250.000`
+`name` | `row['car_model_name']`
+`g_CO2_km` | -
+`vehicle_image` | -
+`make` | first word of `row['car_model_name']`
+`model` | all words after the first word of `row['car_model_name']`
+`wheel_count` | -
+`max_permitted_speed` | -
+`rated_power` | -
+`default_reserve_time` | -
+`return_constraint`| `roundtrip`
+`vehicle_assets`| -
+`default_pricing_plan_id`| `tarif` (current provided only has one pricing plan, might be overwritten in subclasses)
+`pricing_plan_ids`| -
+
+
+### station_information.json
+
+Station information is extracted from `parkings` endpoint.
+
+
+#### Field Mappings
+
+GBFS Field | Mapping
+--- | ---
+`station_id` | `row['id']`
+`name` | `row['name']`
+`short_name` | -
+`lat` | `row['center']['lat']`
+`lon` | `row['center']['lon']`
+`address` | `row['street'] + ' ' + row['street_number']`
```

Comment:
Should we prefix with row['town'] here instead of introducing a "_city" field?

Reply:
https://github.com/mobidata-bw/x2gbfs/pull/75/commits/98b42a3ca048139b204a6b9f89c21f9b5ab8f874